osu.Game/Users/UserPanel.cs

```diff
@@ -179,9 +192,34 @@ public MenuItem[] ContextMenuItems
 
                 bool isUserOnline() => metadataClient?.GetPresence(User.OnlineID) != null;
                 bool canInviteUser() => isUserOnline() && multiplayerClient?.Room?.Users.All(u => u.UserID != User.Id) == true;
+                bool isUserBlocked() => api.Blocks.Any(b => b.TargetID == User.OnlineID);
             }
         }
 
+        private void blockUser(bool block)
```

Comment:
We're going to want to be able to block users from other locations in game in the future. I guess this can exist here for now, but we'll eventually want to reconsider.